### PR TITLE
`Forms` : Strings add missing quantities

### PR DESCRIPTION
**Pull request by kaushikrw**
Related to issue: #[apollo/986](https://devtopia.esri.com/runtime/apollo/issues/986)

### Description:

Adds missing quantities to the string plural resources that will be translated.

### Translation notes

- The locale folder "values-he" should be called "values-iw" instead.
- The locale folder "values-id" should be called "values-in" instead.
- For some languages, some quantities are not relevant. This however, is only a lint warning.

### Pre-merge Checklist

- a [vTest](https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/) Job for this PR has been run
  - [x] link: https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/242/
- Unit and/or integration tests have been added to exercise this PR's logic, and the tests are passing:
  - [ ] Yes
  - [ ] No

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ androidGradlePlugin = "8.3.2"
 androidXBrowser = "1.8.0"
 coilBOM = "2.5.0"
 composeBOM = "2024.10.00"
-androidxCamera = "1.4.0-rc02"
+androidxCamera = "1.4.0"
```
Comment (kaushikrw) [inline gradle/libs.versions.toml:7]: update to the latest stable.

**Comment by kaushikrw** [comment]
The vTest has lint failures related to another component.

**Review by sorenoid — APPROVED** [review]
LGTM. Thank you for getting the lint straightened out.

**Comment by kaushikrw** [comment]
Thanks @sorenoid.

**Pull request merged**